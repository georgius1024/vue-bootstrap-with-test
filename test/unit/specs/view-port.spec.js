import {
  shallowMount,
  createLocalVue
} from '@vue/test-utils'
import VueRouter from 'vue-router'
import Component from '@/components/app/view-port'

const localVue = createLocalVue()
localVue.use(VueRouter)
const router = new VueRouter()


describe('view-port.vue', () => {

  it('should render correct way', () => {
    const wrapper = shallowMount(Component, {
      localVue,
      router
    })
    expect(wrapper).toMatchSnapshot();
  })

})
