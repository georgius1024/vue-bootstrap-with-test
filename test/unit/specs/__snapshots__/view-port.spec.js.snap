// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`view-port.vue should render correct way 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
  "version": 2.5,
}
`;
